### PR TITLE
Removing single lepton paths

### PR DESCRIPTION
**Pull request by OlivierBondu**
note that, as per in-file comment:

> IT IS STRONGLY ADVISED TO HAVE YOUR OWN `triggers.xml` ON THE ANALYSIS SIDE

to make sure you are using the triggers you want to, not the subset the framework default thinks you want :wink:

### CODE REVIEW

**Comment by blinkseb** [comment]
Can you rebase ? :)

**Comment by OlivierBondu** [comment]
oops, my bad, just rebased!

**Comment by OlivierBondu** [comment]
updated: fixed a typo

File: data/triggers.xml [hunk 1]
```diff
@@ -23,23 +21,16 @@
...
-        <path>HLT_IsoMu27_v.*</path>
     </runs>
     <!-- Run 2016 -->
+    <!-- IT IS STRONGLY ADVISED TO HAVE YOUR OWN triggers.xml ON THE ANALYSIS SIDE -->
     <runs from="271036" to="999999">
         <!-- Double mu -->
         <path>HLT_Mu17_TrkIsoVVL_(Tk)?Mu8_TrkIsoVVL_v.*</path>
         <!-- Double ele -->
-        <path>HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v*</path>
+        <path>HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v.*</path>
```
Comment (blinkseb) [inline data/triggers.xml:31]: Nice catch :+1:

**Comment by blinkseb** [comment]
Thanks!

**Pull request merged**